flavours/oauth/README.md

```diff
@@ -0,0 +1,51 @@
+# Dummy OAuth konnector
+
+This konnector is aimed to make development for OAuth konnectors easier. It uses a dedicated cozy instance as OAuth provider.
+
+Typical usage is :
+> From `cozy.tools:8080`, the Dummy OAuth konnctor make an OAuth connection to `cozy.tools:8181`.
+
+## Set up
+
+To be used, this konnector needs a little set up. We assume that there is already a CouchDB running, and a cozy stack with an instance `cozy.tools:8080`.
+
+### Dedicated instance
+
+We first need to run another cozy stack on port 8181 and create a new instance.
```

Comment:
It seems a bit weird to run two cozy-stack processes, instead of just using the same stack with two instances (like cozy.tools:8080 and dummy.test.cozy.tools:8080)

Reply:
You're right. Let me try.

Reply:
That worked. I'll update the PR ASAP.